apps/frontend/src/app/components/insightConfiguration/InsightConfiguration.tsx

```diff
@@ -0,0 +1,147 @@
+                    <TextField
+                        id="name"
+                        label="Navn"
+                        onChange={handleInputChange}
+                        value={insight.name}
+                    />
+                    <Textarea
+                        id="description"
+                        label="Beskrivelse"
+                        onChange={handleInputChange}
+                        value={insight.description}
+                    />
+                    <div className={style.dates}>
+                        <Textarea
+                            id="starts"
+                            label="Startdato"
+                            placeholder="DD-MM-ÅÅÅÅ"
+                            onChange={handleInputChange}
+                            value={insight.starts}
+                        />
+                        <Textarea
+                            id="ends"
+                            label="Sluttdato"
+                            placeholder="DD-MM-ÅÅÅÅ"
+                            onChange={handleInputChange}
+                            value={insight.ends}
+                        />
+                    </div>
+                </div>
+                <div className={style.insightSpecs}>
+                    <div className={style.insightTraits}>
+                        <Label size="medium" spacing>
+                            Kriterier:
+                        </Label>
+                        {insight.traits.map((trait, index) => {
+                            return (
+                                <div
+                                    key={index}
+                                    onClick={() => removeTrait(trait)}>
+                                    {trait.name}
+                                </div>
+                            );
+                        })}
+                        <Button
+                            size="small"
+                            onClick={() => setOpenTraits(true)}>
+                            + Legg til kriterie
+                        </Button>
+                    </div>
+                    <div className={style.insightConsents}>
+                        <Label size="medium" spacing>
+                            Samtykker:
+                        </Label>
+                        {insight.consents.map((consent, index) => {
+                            return (
+                                <div
+                                    key={index}
+                                    onClick={() => removeConsent(consent)}>
+                                    {consent.name}
+                                </div>
+                            );
+                        })}
+                        <Button
+                            size="small"
+                            onClick={() => setOpenConsents(true)}>
+                            + Legg til samtykke
+                        </Button>
+                    </div>
```

Comment:
Here there is a lot going on, i know we have not settled yet on the design, but i feel like some of these can be seperated into seperate components

Reply:
Agreed, the "trait" and "consent" lists should be separated. Will leave a comment for this temporarily, until a more final design has been decided upon.